### PR TITLE
Update vmw-how-access-centos-update-servers.md

### PR DESCRIPTION
**Pull request by suehighmoor**

### CODE REVIEW

**Comment by suehighmoor** [comment]
@cantlep, @simonrowan, can one of you take a quick look through and approve please?

File: articles/vmware/vmw-how-access-centos-update-servers.md [hunk 1]
```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
 
 ### Option 1. UKCloud-managed repository servers
 
-We provide repositories for CentOS 6 on our Elevated OFFICIAL security domain. To access them:
+We provide repositories for CentOS 6 and 7 on our Elevated OFFICIAL security domain. To access them:
```
Comment (cantlep) [inline articles/vmware/vmw-how-access-centos-update-servers.md]: and EPEL (Extra Packages) for both CentOS6 and 7 also.

File: articles/vmware/vmw-how-access-centos-update-servers.md [hunk 2]
```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
...
 
 Create a file called `il3-repos.repo` in `/etc/yum.repos.d`, and populate it with the following:
```
Comment (cantlep) [inline articles/vmware/vmw-how-access-centos-update-servers.md]: Reword to

For CentOS base files, created a file called il3-repos.repo in /etc/yum.repos.d and populate it with the following

File: articles/vmware/vmw-how-access-centos-update-servers.md [hunk 3]
```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
...
```
Comment (cantlep) [inline articles/vmware/vmw-how-access-centos-update-servers.md]: CentOS 6 looks like so:

[base]
name=CentOS-6-Base
baseurl=http://x.x.x.xcentos-base/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6

#released updates
[updates]
name=CentOS-6-Updates
baseurl=http://x.x.x.x/centos-updates/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6

#additional packages that may be useful
[extras]
name=CentOS-6-Extras
baseurl=http://x.x.x.x/centos-extras/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6

File: articles/vmware/vmw-how-access-centos-update-servers.md [hunk 4]
```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
...
 
-    ```
```
Comment (cantlep) [inline articles/vmware/vmw-how-access-centos-update-servers.md]: CentOS7 looks like so:
[base]
name=CentOS-7-Base
baseurl=http://x.x.x.xcentos-7-base/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7

#released updates
[updates]
name=CentOS-7-Updates
baseurl=http://x.x.x.x/centos-7-updates/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7

#additional packages that may be useful
[extras]
name=CentOS-7-Extras
baseurl=http://x.x.x.x/centos-7-extras/
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7

File: articles/vmware/vmw-how-access-centos-update-servers.md [hunk 5]
```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
...
 
 Or use the following command to download the repo file directly from the server:
 
-    ```
-    wget -P /etc/yum.repos.d http://<elevated-public-ip-address>/repos/ukcloud_centos6.repo
-    ```
-
-In the above string, replace `elevated-public-ip-address` with the Elevated OFFICIAL public IP addresses. If you're not sure what this is, contact UKCloud Support.
+```
+wget -P /etc/yum.repos.d http://<elevated-public-ip-address>/repos/ukcloud_centos<x>.repo
```
Comment (cantlep) [inline articles/vmware/vmw-how-access-centos-update-servers.md]: Change this URL to the following based on version:

wget -P /etc/yum.repos.d http://<elevated-public-ip-address>/repos/ukcloud_centos<x>.repo

For CentOS base,updates and extras where X is either 6 or 7
wget -P /etc/yum.repos.d/ http://<elevated-public-ip-address>/customer_repo_files/centos_X_il3.repo

For CentOS EPEL where X is either 6 or 7
wget -P /etc/yum.repos.d/ http://<elevated-public-ip-address>/customer_repo_files/epel_X_il3.repo

File: articles/vmware/vmw-how-access-centos-update-servers.md [hunk 6]
```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
...
+```
 
 Remove all other `*.repo` files in this directory.
```
Comment (cantlep) [inline articles/vmware/vmw-how-access-centos-update-servers.md]: then execute 'yum repolist'

File: articles/vmware/vmw-how-access-centos-update-servers.md [hunk 7]
```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
...
-
-    name=UKCloud CentOS Repository - Extras
-
-    baseurl=http://<elevated-public-ip-address>/centos/centos-extras
-
-    gpgcheck=1
-
-    gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
-    ```
+- `<x>.<y>` is your version of CentOS
```
Comment (cantlep) [inline articles/vmware/vmw-how-access-centos-update-servers.md]: No need for this. We don't specify incremental versions.

File: articles/vmware/vmw-how-access-centos-update-servers.md [hunk 8]
```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
...
```
Comment (cantlep) [inline articles/vmware/vmw-how-access-centos-update-servers.md]: For CentOS EPEL files, created a file called il3-epel.repo in /etc/yum.repos.d and populate it with the following

For CentOS6:

[epel]
name=Extra Packages for Enterprise Linux 6
baseurl=http://x.x.x.x/epel/
enabled=1
metadata_expire=7d
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-6


For CentOS7:

[epel]
name=Extra Packages for Enterprise Linux 7
baseurl=http://x.x.x.x/epel-7/
enabled=1
metadata_expire=7d
gpgcheck=1
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7

File: articles/vmware/vmw-how-access-centos-update-servers.md [hunk 9]
```diff
@@ -31,55 +31,105 @@ UKCloud's Assured OFFICIAL security domain is internet facing, so you need to co
...
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7
+```
+
+To download the repo file directly from the server, use the following command (`<x>` is either 6 or 7 depending on your version of CentOS):
+
+**CentOS base updates and extras:**
+
+```none
+wget -P /etc/yum.repos.d/ http:///<customer_repo_files>/centos_<x>_il3.repo
```
Comment (cantlep) [inline articles/vmware/vmw-how-access-centos-update-servers.md]: Too many /// and missing elevated IP
The URLs here should be 

http://<elevated-public-ip-address>/customer_repo_files/centos_<x>_il3.repo
and
http://<elevated-public-ip-address>/customer_repo_files/epel_<x>_il3.repo

**Pull request merged**